src/main/java/fi/aalto/cs/apluscourses/model/CoursePluginVersion.java

```diff
@@ -0,0 +1,51 @@
+package fi.aalto.cs.apluscourses.model;
+
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
+
+public class CoursePluginVersion {
```

Comment:
This is also potentially confusing in the same way as `getRequiredPluginVersion`. Perhaps this could be renamed to `CourseVersion` and `CURRENT_VERSION` to `DEFAULT_VERSION`?